src/CxxWrap.jl

```diff
@@ -114,6 +110,19 @@ end
 
 to_julia_int(x::Union{CxxSigned,CxxUnsigned}) = reinterpret(julia_int_type(typeof(x)),x)
 
+const CxxNumber = Union{CxxSigned,CxxUnsigned}
+Base.show(io::IO, n::CxxNumber) = show(to_julia_int(n))
```

Comment:
`= show(io, to_julia_int(n))`

Reply:
Thanks, caught it too in the mean time writing the tests :)